.drone.yml

```diff
@@ -85,6 +85,7 @@ steps:
     BINDIR: /usr/local/bin
   commands:
     - make integration
+    - build/osctl-linux-amd64 config generate cluster.local 1.2.3.4,2.3.4.5,3.4.5.6
```

Comment:
Awesome. We may want to `cd` into `/tmp` to avoid dirtying the current dir and making images have the `-dirty` suffix.